service/ui.go

```diff
@@ -54,18 +54,20 @@ type UIServer struct {
 
 // ViewData contains common data that is provided to all Evergreen pages
 type ViewData struct {
-	User                *user.DBUser
-	ProjectData         projectContext
-	Project             model.Project
-	Flashes             []interface{}
-	Banner              string
-	BannerTheme         string
-	Csrf                htmlTemplate.HTML
-	JiraHost            string
-	NewRelic            evergreen.NewRelicConfig
-	IsAdmin             bool
-	NewUILink           string
-	ValidDefaultLoggers []string
+	User                                   *user.DBUser
+	ProjectData                            projectContext
+	Project                                model.Project
+	Flashes                                []interface{}
+	Banner                                 string
+	BannerTheme                            string
+	Csrf                                   htmlTemplate.HTML
+	JiraHost                               string
+	NewRelic                               evergreen.NewRelicConfig
+	IsAdmin                                bool
+	NewUILink                              string
+	ValidDefaultLoggers                    []string
+	ValidDefaultHostAllocatorRoundingRules []string
```

Comment:
I don't think it makes much sense to have this data be sent with every page. It's fine to hardcode the valid values in the UI, especially since we won't be using it for long